src/utilities.js

```diff
@@ -89,12 +89,12 @@ module.exports = class Utilities {
     await this.loadTherapySettings();
   }
   async addUnconfiguredPump() {
-    let statusScreen = await this._testApp.OpenStatusScreen();
```

Comment:
This seems to be another file that is a catch all. Again not sure if it makes sense to rename or restructure, but thought Is should point out that until someone knows what is in `utilities` they probably don't know to look there for these things.

Reply:
yes. left a nice brain dump comment above :)